simulation/src/test/resources/report.md

```diff
@@ -1,7 +1,7 @@
 # Report
 <!-- Run SimulationTest to regenerate this report. -->
 ```
-          all_nodes_500[CONCURRENCY_LIMITER_BLACKLIST_ROUND_ROBIN].txt:	success=78.1%	client_mean=PT2.54955S     	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1562, 500=438}
+          all_nodes_500[CONCURRENCY_LIMITER_BLACKLIST_ROUND_ROBIN].txt:	success=70.6%	client_mean=PT1.7802S      	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1411, 500=589}
```

Comment:
this looks a lot better https://github.com/palantir/dialogue/blob/ckozak/blacklist_signal/simulation/src/test/resources/report.md#all_nodes_500concurrency_limiter_blacklist_round_robin